### PR TITLE
docs: Add HowTo to run connectors with local stack

### PR DESCRIPTION
**Pull request by taratatach**

### CODE REVIEW

**Review by doubleface — CHANGES_REQUESTED** [review]
I think you just need to update mkdocs.yml.tpl with this file

File: src/howTos/dev/run-connectors-on-local-stack.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: How to run connectors with a local stack
+---
+
+Running connectors to test them with a local Cozy stack gives a quick feedback loop but requires a little setup.
```
Comment (Ldoppea) [inline src/howTos/dev/run-connectors-on-local-stack.md:5]:
```suggestion
Running connectors to test them with a local cozy-stack gives a quick feedback loop but requires a little setup.
```

File: src/howTos/dev/run-connectors-on-local-stack.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## Prerequisite
+
+- Have a local stack installed
```
Comment (Ldoppea) [inline src/howTos/dev/run-connectors-on-local-stack.md:9]:
```suggestion
- Have a local cozy-stack installed
```

File: src/howTos/dev/run-connectors-on-local-stack.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+- Have NodeJS installed at the system level
+
+## Configure your local stack
```
Comment (Ldoppea) [inline src/howTos/dev/run-connectors-on-local-stack.md:12]:
```suggestion
## Configure your local cozy-stack
```

File: src/howTos/dev/run-connectors-on-local-stack.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+---
+title: How to run connectors with a local stack
```
Comment (Ldoppea) [inline src/howTos/dev/run-connectors-on-local-stack.md:2]:
```suggestion
title: How to run connectors with a local cozy-stack
```

File: src/howTos/dev/run-connectors-on-local-stack.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+Running connectors to test them with a local Cozy stack gives a quick feedback loop but requires a little setup.
+
+## Prerequisite
+
+- Have a local stack installed
+- Have NodeJS installed at the system level
+
+## Configure your local stack
+
+Your local stack needs to know where to find the connectors runner on your filesystem. To this end, you'll need to [download the script](https://raw.githubusercontent.com/cozy/cozy-stack/master/scripts/konnector-node-run.sh) from the [cozy-stack repository](https://github.com/cozy/cozy-stack) (or clone the entire repository) and then set the path in your `cozy.yaml` file as such:
```
Comment (Ldoppea) [inline src/howTos/dev/run-connectors-on-local-stack.md:14]:
```suggestion
Your local cozy-stack needs to know where to find the connectors runner on your filesystem. To this end, you'll need to [download the script](https://raw.githubusercontent.com/cozy/cozy-stack/master/scripts/konnector-node-run.sh) from the [cozy-stack repository](https://github.com/cozy/cozy-stack) (or clone the entire repository) and then set the path in your `cozy.yaml` file as such:
```

File: src/howTos/dev/run-connectors-on-local-stack.md [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+```yaml
+# This top-level key might already exist if you used the template from the 
+# cozy-stack repository
+konnectors:
+  cmd: <absolute path to konnector-node-run.sh>
+ 
+```
+
+You can now restart your local Cozy stack and install your connector (for more information on how to run a connector from a local directory, check the [connectors tutorial](https://docs.cozy.io/en/tutorials/konnector/save-data/)).
```
Comment (Ldoppea) [inline src/howTos/dev/run-connectors-on-local-stack.md:24]:
```suggestion
You can now restart your local cozy-stack and install your connector (for more information on how to run a connector from a local directory, check the [connectors tutorial](https://docs.cozy.io/en/tutorials/konnector/save-data/)).
```

**Review by Ldoppea — APPROVED** [review]
nit: Just some homogenisation of the `cozy-stack` term needed, but everything else looks good to me

**Review by doubleface — APPROVED** [review]

**Pull request merged**